skxray/core.py

```diff
@@ -1226,3 +1226,44 @@ def multi_tau_lags(multitau_levels, multitau_channels):
     lag_steps = np.append(lag_steps, np.array(lag))
 
     return tot_channels, lag_steps
+
+
+def geometric_series(common_ratio, number_of_images, first_term=1):
```

Comment:
@ericdill All these functions are going to use in XSVS2.
As an example in XSVS they collect 100 images, each take in1ms,
Then they choose ROI and see the photon count probability density  in that ROI for 1ms, 2ms, 4ms, 8s, 16ms, etc
or 1ms, 3ms, 9ms, 27ms,  etc like, 
This will create for 100 images, common ratio=2,  geometric_series =  1, 2, 4, 8, 16, 32, 64